tests/test_deprecation.py

```diff
@@ -0,0 +1,22 @@
+from . import unittest, THIS
+import mock
+import sys
+
+dep_msg = 'bio_pieces will be removed in the future. Please use import bio_bits'
+
+@mock.patch('bio_bits.version.get_release', mock.MagicMock(return_value=1.0))
+@mock.patch.object(sys, 'stderr')
+class TestProjectRename(unittest.TestCase):
+    def test_can_import_bio_pieces(self, mock_serr):
+        import bio_pieces
+        r = bio_pieces.version.get_release()
+        self.assertEqual(1.0, r)
+        actual_msg = mock_serr.write.call_args[0][0]
+        self.assertIn(dep_msg, actual_msg)
+
+    def test_can_import_sub_module(self, mock_serr):
+        from bio_pieces import version
+        r = version.get_release()
+        self.assertEqual(1.0, r)
+        actual_msg = mock_serr.write.call_args[0][0]
```

Comment:
Any thoughts on why this test does not work?
https://travis-ci.org/VDBWRAIR/bio_pieces/jobs/90775465#L841

If I import it from the command line I get the warning
```
$ bin/python -c "from bio_pieces import version"
/media/VD_Research/People/tyghe.vallard/Projects/bio_bits/bio_pieces/__init__.py:7: UserWarning: bio_pieces will be removed in the future. Please use import bio_bits
  'bio_pieces will be removed in the future. Please use import bio_bits'